gr-azure-software-radio/python/blob_sink.py

```diff
@@ -95,6 +86,55 @@ def __init__(self, np_dtype: np.dtype, vlen: int = 1, authentication_method: str
         self.queue = queue.Queue(maxsize=queue_size)
         self.buf = np.zeros((self.block_len*self.item_size, ), dtype=np.byte)
         self.num_buf_bytes = 0
+        self.log = gr.logger(f"gr_log.{self.symbol_name()}")
```

Comment:
Note that prior to this change, log messages werent actually showing up in the console (e.g. in GRC or in the console used to run the flowgraph) and now they do.  However, our tests also show the logs which makes them a bit messy, I couldn't quite figure out how to change the log level as part of the pytest code.